src/Hyperlink/index.jsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import classNames from 'classnames';
+import FontAwesomeStyles from 'font-awesome/css/font-awesome.min.css';
+import PropTypes from 'prop-types';
+import isRequiredIf from 'react-proptype-conditional-require';
+
+function Hyperlink(props) {
+  const {
+    destination,
+    content,
+    target,
+    onClick,
+    externalLinkAlternativeText,
+    externalLinkTitle,
+    ...other
+  } = props;
+
+  let externalLinkIcon;
+
+  if (target === '_blank') {
+    externalLinkIcon = (
+      // Space between content and icon
+      <span>{' '}
+        <span
+          className={classNames(FontAwesomeStyles.fa, FontAwesomeStyles['fa-external-link'])}
+          aria-hidden={false}
+          aria-label={externalLinkAlternativeText}
+          title={externalLinkTitle}
+        />
+      </span>
+    );
+  }
+
+  return (
+    <a
+      href={destination}
+      target={target}
+      onClick={onClick}
+      {...other}
+    >{content}{externalLinkIcon}
+    </a>
+  );
+}
+
+Hyperlink.defaultProps = {
+  target: '_self',
+  onClick: () => {},
```

Comment:
If this is a proper hyperlink, what's the use case for an `onClick`? I'm not sure we need it.

Reply:
(...maybe for event tracking?)

Reply:
That's an interesting point - I guess I was kind've future-proofing here...if you think it's unnecessary I can remove it.

Reply:
Actually, we definitely use `onClick` event handlers for analytics in Marketing so I think we should keep this.